docs/index.md

```diff
@@ -665,10 +665,16 @@ $ mocha
 
 <h2 id="test-duration">Test duration</h2>
 
-Many reporters will display test duration, as well as flagging tests that are slow, as shown here with the "spec" reporter:
+Many reporters will display test duration and flag tests that are slow (default: 75ms), as shown here with the "spec" reporter:
 
 ![test duration](images/reporter-spec-duration.png?withoutEnlargement&resize=920,9999){:class="screenshot"}
 
+There are three levels of test duration:
+
+1. FAST: Tests run under half of the "slow" threshold is considered "FAST". The duration will be shown in green or not shown at all.
+2. REASONABLE (in yellow): Tests duration under threshold but above half of threshold will be shown in yellow.
```

Comment:
MEDIUM: Tests that run above half of threshold (but under the threshold) will show the duration in yellow.